install.sh

```diff
@@ -186,6 +186,9 @@ fi
 # Required for Nvidia GPU support on Linux (package does not exist on macOS)
 [[ $(uname) == "Linux" ]] && conda install --yes pocl-cuda
 
+# Necessary to build mpi4py
+[[ $(uname) == "Linux" ]] && conda install --yes gcc_linux-64
```

Comment:
What happens if you don't have this available? What about macOS? What about Linux+Power9?

Reply:
I found a better workaround that "fixes" the qt package that is causing this issue. https://github.com/conda-forge/qt-feedstock/issues/208 has more context.